### ISSUE TITLE
matomo:install main_url is always set to http://example.com

### ISSUE THREAD

**Issue by chriscroome** [digitalist-se/extratools#11]
When doing a Matomo install with the following command:

```bash
php console --no-ansi -n matomo:install --db-username=stats --db-pass=XXXXXX --db-host=localhost --db-name=stats --first-site-name='wsh.webarchitects.org.uk' --first-site-url='https://wsh.webarchitects.org.uk' --first-user='webarch' --first-user-email='chris@webarchitects.co.uk' --first-user-pass='XXXXXX' --force
```

The `main_url` for the first site is set to `http://example.com` despite it being provided on the command line:

```mysql

MariaDB [stats]> select idsite,name,main_url from site;
+--------+--------------------------+--------------------+
| idsite | name                     | main_url           |
+--------+--------------------------+--------------------+
|      1 | wsh.webarchitects.org.uk | http://example.com |
+--------+--------------------------+--------------------+
1 row in set (0.001 sec)
```

Is this caused by a mistake of mine? Or is there an issue with the `matomo:install` command?

**Comment by mikkeschiren** [comment]
I'm sure it is a misstake of mine - will check it out in the coming days.

**Comment by chriscroome** [comment]
Cheers @mikkeschiren, that is great, I understand that I shouldn't be using this for production use and I'm [working around the issue](https://git.coop/webarch/matomo/commit/ad9a3288e2b7ec6e6069f0b648021d1ba241156a#c826a895ebc5a897a8ac4016388cbc4dcbaa8f08_97_97) so I'm happy to see it fixed when you have some spare time.

**Comment by mikkeschiren** [comment]
This should be fixed in the latest release (0.1.0-alpha9)
